app/assets/javascripts/admin/components/admin-editable-field.js.es6

```diff
@@ -0,0 +1,23 @@
+export default Ember.Component.extend({
+  tagName: "",
+
+  buffer: "",
+  editing: false,
+
+  init() {
+    this._super();
+    this.set("editing", false);
```

Comment:
is this necessary ? I'm unsure

Reply:
Yes, it is. `editing` is coming from the controller and the controller's state is not reset.